src/Websites/Websites/help/Import-AzWebAppKeyVaultCertificate.md

```diff
@@ -73,7 +73,7 @@ Accept wildcard characters: False
 ```
 
 ### -KeyVaultName
-The name of the keyvault.
+The name of the keyvault or Id of the KeyVault.
```

Comment:
The description should be added in the definition of parameter `KeyVaultName` as well.